src/main/java/modify_task_use_case/ModifyTaskInteractor.java

```diff
@@ -20,21 +20,30 @@ public ModifyTaskInteractor(
         this.dsGateway = dsGateway;
     }
 
+    /**
+     * @param inputData contains the dayIndex which contains the Task to be modified,
+     * the existing title of the Task, and the new title of the task.
+     *
+     * @return A ModifyTaskOutputData instance which contains title, dayIndex, errorMessage
+     * and isSuccessfully modified attributes, such that the latter two indicate whether the
+     * use case was successful.
+     */
     @Override
     public ModifyTaskOutputData modifyTask(ModifyTaskInputData inputData) {
-        if (dsGateway.taskExistsByTitle(inputData.getNewTitle(), inputData.getDayID())) {
+        ModifyTaskOutputData outputData = new ModifyTaskOutputData(
+                inputData.getNewTitle(), inputData.getDayIndex()
+        );
+        if (dsGateway.taskExistsByTitle(inputData.getNewTitle(), inputData.getDayIndex())) {
             return outputBoundary.prepareFailView(
-                    "Task with name: " + inputData.getTitle() + " already exists for day " + inputData.getDayID()
+                    outputData,
+                    "Task with name: " + inputData.getNewTitle() + " already exists for day " + inputData.getDayIndex()
```

Comment:
This wording doesn't make any sense as an error message. For example "Task with name: clean dishes already exists for day 1" doesn't make sense to the user. You should add some quotations around the task name and also use the string form of the day rather than an index. I would recommend something along the lines of "Task with name 'clean dishes' already exists on Monday"

Reply:
Noted. I will add quotations around the name of the Task and explicitly state the name of the day in my error message.